client/components/waveformPlayer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Comment from './comment.js';
+
+export default class WaveformPlayer extends React.Component {
+  constructor(props) {
+    super(props);
+  };
+  changeTimeToMinutesAndSeconds(timestamp) {
```

Comment:
You have very clear variable and function names!  Keep it up!

Reply:
Thanks!